build_tools/buildkite/scripts/wait_for_pipeline_success.py

```diff
@@ -250,6 +87,11 @@ def main(args):
     sys.exit(1)
   print(f"Build completed successfully: {linkify(url)}")
 
+  if args.output_build_json:
+    build = bk.get_build_by_number(build_number)
```

Comment:
It seems redundant to fetch this again. We can make wait_for_build return the full build instead of just the state

Reply:
Done.